### PR TITLE
directanswers: style string text return option

### PR DESCRIPTION
**Pull request by apokellypse**

### CODE REVIEW

**Comment by apokellypse** [comment]
Forgot to Select the correct version branch. Closing

**Comment by vramesh1** [comment]
the latest changes on v0.9.3 will need to be pulled into your branch.
could you rebase your branch on top of v0.9.3?

**Review by Delerme — DISMISSED** [review]
You can use the 'update branch' button below to merge in v0.9.3 changes, since we use the 'squash & merge' option.

Also for visual changes send a screenshot or simulacrum demo via slack

File: src/ui/sass/modules/_DirectAnswer.scss [hunk 1]
```diff
@@ -59,6 +73,10 @@
 
   &-content
   {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 60px;
```
Comment (creotutar) [inline src/ui/sass/modules/_DirectAnswer.scss]: nit: do we want this to relative to `$base-spacing-sm`

File: src/ui/sass/modules/_DirectAnswer.scss [hunk 2]
```diff
@@ -74,6 +92,7 @@
   {
     display: flex;
     justify-content: flex-end;
+    height: 24px;
```
Comment (creotutar) [inline src/ui/sass/modules/_DirectAnswer.scss]: nit: do we want this to relative to `$base-spacing-sm`

File: src/ui/sass/modules/_DirectAnswer.scss [hunk 3]
```diff
@@ -84,6 +103,7 @@
 
   &-footerText
   {
+    margin-right: 16px;
```
Comment (creotutar) [inline src/ui/sass/modules/_DirectAnswer.scss]: nit: do we want this to relative to `$base-spacing`

File: src/ui/sass/modules/_DirectAnswer.scss [hunk 4]
```diff
@@ -162,4 +183,10 @@
     display: inline-flex;
     align-items: center;
   }
+
+  &-viewMoreIcon
+  {
+    $line-height: $direct-answer-view-details-line-height;
+    margin-left: 4px;
```
Comment (creotutar) [inline src/ui/sass/modules/_DirectAnswer.scss:190]: nit: do we want this to relative to `$base-spacing`

File: src/ui/sass/modules/_DirectAnswer.scss [hunk 5]
```diff
@@ -15,20 +15,21 @@
...
-  $direct-answer-footer-font-size: 13.5px !default;
+  $direct-answer-footer-font-size: $font-size-md !default;
   $direct-answer-footer-line-height: $line-height-md !default;
   $direct-answer-footer-font-weight: $font-weight-normal !default;
-  $direct-answer-footer-color: $color-text-primary !default;
+  $direct-answer-footer-color: $color-text-secondary !default;
 
   &-title
   {
+    height: 26px;
```
Comment (creotutar) [inline src/ui/sass/modules/_DirectAnswer.scss]: we usually avoid giving explicit heights to things that arent images b/c some divs can expand to two lines of text. is this the case here? maybe we want to use padding to achieve this?

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 1 above)
Comment (creotutar) [reply src/ui/sass/modules/_DirectAnswer.scss]: same height comment here, are we guaranteed the content will never exceed 60px in the case of wrapping text / number of lines for a direct answer?

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 2 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss]: the footer height seems independent of horizontal spacing, so I don't think so in this case. will add as base variable $module-footer-height so it is more visible.

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 1 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss]: good point, this won't handle wrapping. removing for now

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 3 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss]: Yes, we do. Will fix

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 4 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss:190]: don't think so; we want icons to be close to their action text regardless of what may eventually override base-spacing

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 5 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss]: we can definitely use padding or margin to achieve this!

**Review by creotutar — DISMISSED** [review]
🎺 🎺 🎺 LGTM

File: src/ui/sass/_fonts.scss [hunk 6]
```diff
@@ -19,3 +19,4 @@ $line-height-lg: 1.5;
 $line-height-xlg: 5/3;
 $line-height-xxlg: 1.7;
 
+$font-family: system-ui,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif !default;
```
Comment (benmcginnis) [inline src/ui/sass/_fonts.scss:22]: ❤️

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 4 above)
Comment (benmcginnis) [reply src/ui/sass/modules/_DirectAnswer.scss:190]: I would recommend using the `ex` measurement as that is close to a single space of the font in question.  Usually `0.8ex` is pretty close to two spaces for any given font

**Review by benmcginnis — APPROVED** [review]

File: src/ui/sass/_fonts.scss (hunk 6 above)
Comment (apokellypse) [reply src/ui/sass/_fonts.scss:22]: props to Julian for giving me the font families and their orders!

File: src/ui/sass/modules/_DirectAnswer.scss (hunk 4 above)
Comment (apokellypse) [reply src/ui/sass/modules/_DirectAnswer.scss:190]: what do you think about using `em`? I looked into it and saw that `ex` is relative to the height of the font, while `em` is relative to the width

**Pull request merged**